ff-cl-gen/README.md

```diff
@@ -1,18 +1,19 @@
-# ff-cl-gen [![Crates.io](https://img.shields.io/crates/v/ff-cl-gen.svg)](https://crates.io/crates/ff-cl-gen)
+# ff-cl-gen
 
-OpenCL code generator for finite-field arithmetic over prime fields constructed with Rust [ff](https://github.com/filecoin-project/ff) library.
+> This is a fork of the filecoin [bellperson](https://github.com/filecoin-project/ff-cl-gen) library.
+
+OpenCL code generator for finite-field arithmetic over prime fields constructed with Rust [ginger-lib](https://github.com/HorizenOfficial/ginger-lib) library.
 
 Notes:
  - Limbs are 32/64-bit long, by your choice.
  - The library assumes that the most significant bit of your prime-field is unset. This allows for cheap reductions.
 
 ## Usage
 
-Generating OpenCL codes for Bls12-381 Fr elements:
+Generating OpenCL codes for BN-382 Fr elements:
```

Comment:
Only for BN-382 ?

Reply:
This is just an example. I show here how to construct code for GPU for BN-382. Instead of it any other supported curve can be used